scripts/create-conda-recipe.py

```diff
@@ -135,12 +174,24 @@ def ignore_aliases(self, data):
 }
 
 
+#####################################
+# Write the recipe to conda/meta.yaml
```

Comment:
```suggestion
# Write the recipe to conda/recipe.yaml
```

Reply:
The recipe file is called `meta.yaml` in conda
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html

Reply:
But the name of the file where u are writing is wrong then?

Reply:
Line 196

Reply:
Oh the name of the file was wrong. Fixed now, thanks